saleor/core/utils/filters.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import unicode_literals
+
+
+def get_sort_by_choices(filter):
+    return [(choice[0], choice[1].lower()) for choice in
+            filter.filters['sort_by'].field.choices[1::2]]
```

Comment:
You should not override built in functions, please rename `filter` to something else

Reply:
Any suggestions? I wanted to change it but...

Reply:
We could name it `filter_set` after the class from `django-filters`.